### DISCUSSION TITLE
.NET environment variable differences

### DISCUSSION THREAD

**Discussion by MisinformedDNA** [actions/virtual-environments#7209] [General questions] [answered]
The .NET scripts for [Linux](https://github.com/actions/runner-images/blob/main/images/linux/scripts/installers/dotnetcore-sdk.sh) and [MacOS](https://github.com/actions/runner-images/blob/main/images/macos/provision/core/dotnet.sh) set the environment variable `DOTNET_CLI_TELEMETRY_OPTOUT=1`

The .NET script for [Windows](https://github.com/actions/runner-images/blob/main/images/win/scripts/Installers/Install-DotnetSDK.ps1) sets `DOTNET_MULTILEVEL_LOOKUP=0`

Should all the scripts set the same environment variables?

Also, is there a reason `DOTNET_NOLOGO=0` isn't set?

**Comment by vpolikarpov-akvelon** [comment, marked as answer]
Fixed via https://github.com/actions/runner-images/pull/7400
